.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -78,15 +79,29 @@ steps:
 - task: VisualStudioTestPlatformInstaller@1
   displayName: Ensure VSTest Platform
 
+- ${{ if eq(parameters.enableCaching, 'true') }}:
```

Comment:
fwiw: if you set the type of the parameter `enableCaching` to `boolean`, you can use `true` instead of `'true'`. type safety!

this will require you to change the parameters block format from map/object to array

```yaml
parameters:
  - name: additionalBuildArguments
    type: string
    default: ''
  - name: enableCaching
    type: boolean
    default: false
```

Reply:
TIL